parastell/nwl_utils.py

```diff
@@ -274,40 +305,61 @@ def nwl_plot(
     """
     tor_ext = np.deg2rad(tor_ext)
     pol_ext = np.deg2rad(pol_ext)
-    
+
     coords = extract_coords(source_file)
 
     if num_crossings is not None:
         coords = coords[0:num_crossings]
-    
-    # Load plasma equilibrium data
+
+    if num_threads is None:
+        num_threads = cpu_count()
+
     vmec = read_vmec.VMECData(plas_eq)
 
-    phi_coords, theta_coords = flux_coords(vmec, wall_s, coords)
+    phi_coords = []
+    theta_coords = []
+
+    # split up the work to avoid memory issues
+    iterations = 1
+
+    if chunk_size is not None:
+        iterations = math.ceil(len(coords) / chunk_size)
+    else:
+        chunk_size = len(coords)
```

Comment:
Doesn't this do the same thing?
```suggestion
    if chunk_size is None:
        chunk_size = len(coords)

    iterations = math.ceil(len(coords) / chunk_size)
```

Reply:
Why not call this `chunks` instead of `iterations` and then `for chunk in range(chunks):`